colorpicker: add page (#6098)
tldr-pages/tldr: pages/common/colorpicker.md

# colorpicker

> A minimalist X11 colorpicker.
> More information: <https://github.com/ym1234/colorpicker>.

> - Launch colorpicker and print the HEX and RGB value of each clicked pixel to stdout:

Author: We should maybe somehow include that any mouse action excipt left click exits the program. This thing is almost harder to exit than vim.

Reviewer: `:x`
`:ZZ`
`q`
`q!`

I joke of course. Vim to total beginners is odd, me included.

Maybe we could have in the description, `Press ___ to exit`

Author: I think it would be best to include `> Any mouse gesture except left click will exit the program` in the command description.

Reviewer 2: Resolved as the command's description has been updated.

`colorpicker`

- Only print the color of one clicked pixel and exit:

`colorpicker -o`

- Print the color of each clicked pixel and quit on keypress:

`colorpicker -q`

- Only print the RGB value:

`colorpicker -d`

- Only print the HEX value:

> `colorpicker -r`

Reviewer: ```suggestion
`colorpicker -r`

```

# colorpicker

> A minimalist X11 colorpicker.
> Any mouse gesture except left click will exit the program.
> More information: <https://github.com/ym1234/colorpicker>.

> - Launch colorpicker and print the HEX and RGB value of each clicked pixel to stdout:

Reviewer 3: ```suggestion
- Launch colorpicker and print the hexadecimal and RGB value of each clicked pixel to stdout:
```

`colorpicker`

> - Only print the color of one clicked pixel and exit:

Reviewer 2: Perhaps this will read better?
```suggestion
- Only print the color of one clicked pixel and then exit:
```

`colorpicker --one-shot`

> - Print the color of each clicked pixel and quit on keypress:

Reviewer 2: Perhaps this will read better?
```suggestion
- Print the color of each clicked pixel and quit when a key is pressed:
```

`colorpicker --quit-on-keypress`

- Only print the RGB value:

`colorpicker --rgb`

> - Only print the HEX value:

Reviewer 3: ```suggestion
- Only print the hexadecimal value:
```
